src/Http/Authentication.Abstractions/src/AuthenticationScheme.cs

```diff
@@ -50,6 +51,7 @@ public AuthenticationScheme(string name, string? displayName, Type handlerType)
         /// <summary>
         /// The <see cref="IAuthenticationHandler"/> type that handles this scheme.
         /// </summary>
+        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors)]
```

Comment:
Does the attribute do anything when the property isn't settable?

Reply:
It's tracked via the constructor assignment.